imap_processing/lo/tests/test_science_direct_events.py

```diff
@@ -0,0 +1,290 @@
+import bitstring
+
+from imap_processing.lo.l0.science_direct_events import ScienceDirectEvents
+
+# TODO: These unit tests don't call the public function for the ScienceDirectEvents
+# class and are also only able to run if the contects of __init__ are commented out
+# because I currently don't have any compressed DE data. As a workaround, I am
+# directly setting the DATA attribute and calling and testing the private methods
+# as needed. These tests will be updated when data becomes available. When that happens
+# the need for the bitstring import will also go away.
+
+
+def test_find_decompression_case():
+    # Arrange
+    de = ScienceDirectEvents("fake_packet", "0", "fakepacketname")
+    de.DATA = bitstring.Bits(bin="000100010101")
+    case_number_expected = 1
+
+    # Act
+    de._find_decompression_case()
+    case_number_true = de.case_number
+
+    # Assert
+    assert case_number_true == case_number_expected
```

Comment:
I find the `true == xyz` hard to comprehend. IMO this is shorter and more readable.
```suggestion
    # Assert
    assert de.case_number == case_number_expected
```